src/unity/Assets/Crest/Scripts/LodData/ReadbackLodData.cs

```diff
@@ -50,7 +50,6 @@ private void Update()
         public void UpdateReadback(Camera cam, LodTransform.RenderData renderData)
         {
             // queue pattern inspired by: https://github.com/keijiro/AsyncCaptureTest
-
```

Comment:
perhaps revert this whitespace if you dont mind, the two lines were supposed to be separated, even if it only makes sense to me!!

Reply:
Will do! Sorry, I  didn't even realise I did that.